File: app/src/main/java/com/ludo/todoapp/list/Task.java
package com.ludo.todoapp.list;

import org.jetbrains.annotations.NotNull;

public class Task {
    public String id;
    public String title;
    public String description = "default";

    public Task(String id, String title, String description) {
        this.id = id;
        this.title = title;
        this.description = description;
    }

    public Task(String id, String title) {
        this.id = id;
        this.title = title;
    }

}

File: app/src/main/java/com/ludo/todoapp/list/Task.kt
package com.ludo.todoapp.list

import kotlinx.serialization.SerialName
import java.io.Serializable


@kotlinx.serialization.Serializable
data class Task(
    @SerialName("id")
    var id: String,
    @SerialName("content")
    var title: String,
    @SerialName("description")
    var description: String = "default"
) : Serializable
